benchmarks/fixture/app.tsx

```
@@ -0,0 +1,278 @@
+import { createRoot } from 'react-dom/client'
+import { Tooltip as TooltipV5 } from '../../docs/node_modules/react-tooltip/dist/react-tooltip.mjs'
+import { Tooltip as TooltipV6 } from '../../dist/react-tooltip.esm.js'
+
+type BenchmarkVersion = 'v5' | 'v6'
+
+type RenderMode = 'shared'
+
+type FixtureState = {
+  version: BenchmarkVersion
+  count: number
+  renderMode: RenderMode
```

ESLint

[error] 'renderMode' PropType is defined but prop is never used

(react/no-unused-prop-types)